app/models/project_membership.rb

```diff
@@ -7,4 +7,6 @@ class ProjectMembership < ApplicationRecord
   belongs_to :user
 
   enumerize :role, in: ROLES, predicates: true
+
+  validates :user, :project presence: true
```

Comment:
Redundant. `belongs_to` check record to nil.

Reply:
This PR is still WIP. 
And on the lesson we temporarily added this line because in `db/schema.rb` project_membership's references don't have `null: false` property